tests/Unit/UtilizationMetricsApisTest.php

```diff
@@ -0,0 +1,436 @@
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use App\Models\User;
+
+use function Pest\Laravel\get;
+
+use AdvisingApp\Ai\Models\Prompt;
+use AdvisingApp\Form\Models\Form;
+use AdvisingApp\Task\Models\Task;
+use App\Settings\LicenseSettings;
+use AdvisingApp\Ai\Models\AiThread;
+use AdvisingApp\Alert\Models\Alert;
+use AdvisingApp\Ai\Models\PromptUse;
+use AdvisingApp\Survey\Models\Survey;
+use AdvisingApp\Segment\Models\Segment;
+use App\Http\Middleware\CheckOlympusKey;
+use AdvisingApp\Campaign\Models\Campaign;
+use AdvisingApp\Prospect\Models\Prospect;
+use AdvisingApp\Form\Models\FormSubmission;
+use AdvisingApp\MeetingCenter\Models\Event;
+
+use function Pest\Laravel\withoutMiddleware;
+
+use AdvisingApp\Report\Enums\TrackedEventType;
+use AdvisingApp\Campaign\Models\CampaignAction;
+use AdvisingApp\Survey\Models\SurveySubmission;
+use AdvisingApp\Authorization\Enums\LicenseType;
+use AdvisingApp\Report\Models\TrackedEventCount;
+use AdvisingApp\StudentDataModel\Models\Student;
+use AdvisingApp\ResourceHub\Models\ResourceHubArticle;
+
+beforeEach(function () {
+    withoutMiddleware(CheckOlympusKey::class);
+});
+
+it('checks the API returns users', function () {
+    $randomRecords = random_int(1, 10);
+
+    User::factory()->count($randomRecords)->create();
+
+    $softDeleteUser = User::factory()->create();
+    $softDeleteUser->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['users'])->toBe($randomRecords);
+});
+
+it('checks the API returns AI Users', function () {
+    $randomRecords = random_int(1, 10);
+
+    $licenseSettings = app(LicenseSettings::class);
+
+    $licenseSettings->data->limits->conversationalAiSeats = $randomRecords + 1;
+    $licenseSettings->save();
+
+    User::factory()->count($randomRecords)->licensed([LicenseType::ConversationalAi])->create();
+
+    $softDeleteAiUser = User::factory()->licensed([LicenseType::ConversationalAi])->create();
+    $softDeleteAiUser->delete();
+
+    $response = get('/api/utilization-metrics');
```

Comment:
We should be using the route name here, not calling the path directly.

Reply:
Done